arangod/Aql/DependencyProxy.cpp

```diff
@@ -166,9 +166,7 @@ DependencyProxy<blockPassthrough>::fetchBlock(size_t atMost) {
 
   TRI_ASSERT(!_blockQueue.empty());
 
-  ExecutionState state;
-  SharedAqlItemBlockPtr block;
-  std::tie(state, block) = _blockQueue.front();
+  auto&& [state, block] = _blockQueue.front();
   _blockQueue.pop_front();
```

Comment:
This looks invalid. You can maybe do something like
```
auto [state, block] = std::move( _blockQueue.front());
```
though.

Reply:
Ok, I adjustedt this.